__tests__/topics.test.js

```diff
@@ -52,12 +50,51 @@ describe("Topics api testing", () => {
         expect(topics).toHaveLength(3);
 
         expect(topics[0].slug).toBe("mitch");
-        expect(topics[1].slug).toBe('cats');
-        expect(topics[2].slug).toBe('paper');
+        expect(topics[1].slug).toBe("cats");
+        expect(topics[2].slug).toBe("paper");
 
         expect(topics[0].description).toBe("The man, the Mitch, the legend");
-        expect(topics[1].description).toBe('Not dogs');
```

Comment:
I dont understand why all these changes have been added, be carful when you apply for listing to avoid these kind of wholesale code changes